compiler-rt/lib/scudo/standalone/tests/secondary_test.cpp

```diff
@@ -336,6 +337,37 @@ TEST_F(MapAllocatorCacheTest, CacheOrder) {
     MemMap.unmap();
 }
 
+TEST_F(MapAllocatorCacheTest, PartialChunkHeuristicRetrievalTest) {
+  const scudo::uptr MaxUnusedCacheBytes = scudo::MaxUnusedCachePages * PageSize;
+  const scudo::uptr FragmentedBytes =
+      MaxUnusedCacheBytes +
+      scudo::roundDown(scudo::CachedBlock::ReleaseMemoryUpperBound, PageSize);
```

Comment:
I think that `scudo::CachedBlock::ReleaseMemoryUpperBound` should always align to `PageSize` so maybe we can remove the roundDown()